boards/xtensa/m5stack_stamps3/Kconfig.defconfig

```diff
@@ -0,0 +1,26 @@
+# M5Stack M5Dial board configuration
+# Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_M5STACK_STAMPS3
+
+config BOARD
+	default "m5stack_stamps3"
+	depends on BOARD_M5STACK_STAMPS3
+
+config ENTROPY_GENERATOR
+	default y
+
+config HEAP_MEM_POOL_SIZE
```

Comment:
This config has been updated. Please, change it to `config HEAP_MEM_POOL_ADD_SIZE_BOARD`

Reply:
Fixed in [38f0a59](https://github.com/zephyrproject-rtos/zephyr/commit/38f0a59c9701f755f1b41c0489c642aeb2e0bc7b)